tests/e2e/pageobjects/ide/Editor.ts

```diff
@@ -332,19 +332,33 @@ export class Editor {
         await this.driverHelper.waitDisappearanceWithTimeout(errorInLineLocator, timeout);
     }
 
+    async waitWarningInLine(lineNumber: number, timeout: number = TimeoutConstants.TS_ERROR_HIGHLIGHTING_TIMEOUT) {
```

Comment:
Does it mean "InLineWarning" ?

Reply:
It has the same meaning as here https://github.com/eclipse/che/blob/master/tests/e2e/pageobjects/ide/Editor.ts#L321

Reply:
I see, thank you for explanation.